conda/plugins/manager.py

```diff
@@ -366,18 +360,33 @@ def get_virtual_packages(self) -> tuple[CondaVirtualPackage, ...]:
     def get_reporter_backends(self) -> tuple[CondaReporterBackend, ...]:
         return tuple(self.get_hook_results("reporter_backends"))
 
-    def get_reporter_backend(self, name: str) -> CondaReporterBackend | None:
-        for reporter_backend in self.get_reporter_backends():
+    def get_reporter_backend(self, name: str) -> CondaReporterBackend:
+        """
+        Attempts to find a reporter backend while providing a fallback option if it is
+        not found.
+
+        This method must return a valid ``CondaReporterBackend`` object or else it will
+        raise an exception.
+        """
+        reporter_backend_objs = self.get_reporter_backends()
+
+        for reporter_backend in reporter_backend_objs:
             if reporter_backend.name == name:
                 return reporter_backend
 
-    def get_reporter_outputs(self) -> tuple[CondaReporterOutput, ...]:
-        return tuple(self.get_hook_results("reporter_outputs"))
+        log.warning(
+            f'Unable to find reporter backend: "{name}"; '
+            f'falling back to using "{DEFAULT_CONSOLE_REPORTER_BACKEND}"'
+        )
 
-    def get_reporter_output(self, name: str) -> CondaReporterOutput | None:
-        for reporter_output in self.get_reporter_outputs():
-            if reporter_output.name == name:
-                return reporter_output
+        for reporter_backend in reporter_backend_objs:
+            if reporter_backend.name == DEFAULT_CONSOLE_REPORTER_BACKEND:
+                return reporter_backend
+
+        raise CondaError(
+            "There are no available reporter backends to render output. This conda installation"
+            " is most likely corrupt and requires a re-installation."
+        )
```

Comment:
As mentioned above, that's not a valid outcome of this method and doesn't provide meaningful help to users. Conda essentially _has_ to ship a default console reporter backend that works, similar to the solver.